cmd/kubelet/app/server.go

```diff
@@ -166,7 +167,7 @@ func NewKubeletServer() *KubeletServer {
 		HealthzPort:                 10248,
 		HealthzBindAddress:          util.IP(net.ParseIP("127.0.0.1")),
 		RegisterNode:                true, // will be ignored if no apiserver is configured
-		OOMScoreAdj:                 -900,
+		OOMScoreAdj:                 -1000,
```

Comment:
I do see the point that Anana brings up of having many small guaranteed tasks and the Kubelet be "larger". It is mostly unlikely and the max pods does keep it from happening most of the time as well.

-1000 is "special" and I don't think we want Kubelet to be special in that way.